conference/utils.py

```diff
@@ -14,6 +7,17 @@
 import tempfile
 from collections import defaultdict
 
+from django.conf import settings as dsettings
+from django.core.mail import send_mail as real_send_mail
+from django.core.urlresolvers import reverse
+
+from conference import settings
+from conference.models import VotoTalk, EventTrack
+# TODO: Cleanup conference.tools; the conference module had separate file and directory
+# both called utils which were causing issues resolving dependencies. The
+# directory has been renamed to conference.tools
+from conference.tools import *
```

Comment:
why the '*'?

Reply:
I've removed the `conference.tools` files that were not imported anywhere - the remaining ones are not imported directly, just referred to by their path, so I've updated that. With those changes, the changes to `conference.utils` were no longer necessary and have also been removed.